embedding/cpp/flutter_app.cc

```diff
@@ -177,16 +177,20 @@ void FlutterApp::ParseEngineArgs() {
                         std::string(app_id) + ".rpm");
   free(app_id);
 
-  std::ifstream file(temp_path);
-  if (!file.is_open()) {
+  auto file = fopen(temp_path.c_str(), "r");
+  if (!file) {
+    TizenLog::Error("file is closed");
```

Comment:
Maybe add a path to this log to point exact file which opening failed? Also probably "Failed to open file xxxx" will be more accurate.

Reply:
I would consider removing this log, because this file exists only on first run.

Reply:
@rwalczyna is right. Please keep everything in sync with the C# implementation:

https://github.com/flutter-tizen/flutter-tizen/blob/20007aa24f83ff21ff3f5314410f16e448b0d75c/embedding/csharp/FlutterApplication.cs#L74-L96